pr_testing/test_multiple_prs.sh

```diff
@@ -836,6 +836,7 @@ if [ "X$DO_SHORT_MATRIX" = Xtrue -a "X$BUILD_OK" = Xtrue -a "$ONLY_FIREWORKS" =
     # MATRIX_TIMEOUT is set by jenkins
     dateBefore=$(date +"%s")
     [ $(runTheMatrix.py --help | grep 'job-reports' | wc -l) -gt 0 ] && EXTRA_MATRIX_ARGS="--job-reports $EXTRA_MATRIX_ARGS"
+    [ $(runTheMatrix.py --help | grep 'command' | wc -l) -gt 0 ] && EXTRA_MATRIX_ARGS="--command '--customise Validation/Performance/TimeMemoryJobReport.customiseWithTimeMemoryJobReport' $EXTRA_MATRIX_ARGS"
```

Comment:
@fabiocos , we should add this only for the releases where Validation/Performance/python/TimeMemoryJobReport.py exists otherwise if will break older release cycles.... right?

Reply:
@smuzaffar yes, of course. On the other hand it is such a simple and backward compatible fragment that could be easily backported. Just I am not so sure it is worthwhile. Having it from 10_6_X onwards would be ok...

@slava77 In the log files there is the only addition of the Time report minimal summary (no per module breakdown). In the JobReport there is the same information, plus the Memory service information, which may be useful for a first level monitoring. One may of course do dedicated checks in standalone runs with whatever else tool.